### PR TITLE
Fixed animepahe

### PR DESCRIPTION
**Pull request by Blatzar**
Fixed animepahe with the help of jsbeautifier, it's not necessary as you can try to deobfuscate the javascript with less LOC or just brute force all possible tokens (8-10), but this was the easiest and cleanest solution.

### CODE REVIEW

File: anime_downloader/extractors/kwik.py [hunk 1]
```diff
@@ -15,26 +15,23 @@ class Kwik(BaseExtractor):
...
         # Kwik servers don't have direct link access you need to be referred
         # from somewhere, I will just use the url itself.
 
         download_url = self.url.replace('kwik.cx/e/', 'kwik.cx/f/')
-
         kwik_text = helpers.get(download_url, referer=download_url).text
-        post_url, token = source_parts_re.search(kwik_text).group(1, 2)
+        deobfuscated_text = (jsbeautifier.beautify(kwik_text))
+
+        regex_all = (r"var _[\W\w]{5}=\"[\W\w]*?\"")
```
Comment (29ayush) [inline anime_downloader/extractors/kwik.py]: regex= (r"var _[\W\w]{5}=\"([\W\w]*?)\"")
token = re.search(regex,deobfuscated_text).group(1)[::-1]

Comment (Blatzar) [reply anime_downloader/extractors/kwik.py]: Smart!
Fixed it

File: anime_downloader/extractors/kwik.py [hunk 2]
```diff
@@ -15,26 +15,20 @@ class Kwik(BaseExtractor):
...
-        # than me can work on that for now I will add the pattern I observed
-
-        # alternatively you can pattern match on `src` for stream_url part
-        source_parts_re = re.compile(r'action=\"([^"]+)\".*value=\"([^"]+)\".*Click Here to Download',
-                                     re.DOTALL)
-
         # Kwik servers don't have direct link access you need to be referred
         # from somewhere, I will just use the url itself.
 
         download_url = self.url.replace('kwik.cx/e/', 'kwik.cx/f/')
```
Comment (29ayush) [inline anime_downloader/extractors/kwik.py]: nit: download_url -> site_url.
download_url = site_url.replace('kwik.cx/f/','kwik.cx/d/')

no need to inline so much.

**Comment by ngomile** [comment]
@Blatzar I just wanted to ask but is your kwik fix still working, I am not necessarily trying to use it but I am attempting to access kwik just by using requests or cfscrape and it seems to be blocking me out even when I put the referer.

**Comment by Blatzar** [comment]
> @Blatzar I just wanted to ask but is your kwik fix still working, I am not necessarily trying to use it but I am attempting to access kwik just by using requests or cfscrape and it seems to be blocking me out even when I put the referer.

Kwik uses cloudflare which is currently broken

**Comment by ngomile** [comment]
All right then, waiting game.

**Comment by Blatzar** [comment]
Outdated and doesn't work anymore

**Pull request closed without merging**